test/package-sources-bootc/microshift-gitops.repo

```diff
@@ -0,0 +1,10 @@
+[microshift-gitops]
+name = Red Hat OpenShift GitOps 1.16 for RHEL 9 x86_64 (RPMs)
+baseurl = https://cdn.redhat.com/content/dist/layered/rhel9/{{ .Env.UNAME_M }}/gitops/1.16/os
```

Comment:
Can you please move the gitops version `1.16` into the `common_versions.sh` script?
It would be easier to update the version in the future, as soon as a new gitops version is available.